README.rst

```diff
@@ -36,29 +36,29 @@ Why LISA
 Documents
 ---------
 
--  `Quick start <docs/quick_start.html>`__
--  `Run tests <docs/run_test/run.html>`__
--  `Microsoft tests <docs/run_test/microsoft_tests.html>`__
--  `Write test cases in LISA <docs/write_test/write_case.html>`__
--  `Command line reference <docs/run_test/command_line.html>`__
--  `Runbook reference <docs/run_test/runbook.html>`__
--  `Extend and customize LISA <docs/write_test/extension.html>`__
+-  `Quick start <https://linux-integration-services-automation-lisa.readthedocs.io/en/latest/quick_start.html>`__
```

Comment:
It should avoid to use the absolute links. What's problem here? Is there any other way?

Reply:
My thoughts were if we have a specific website for all docs, we do not need bother showing things on GitHub (so we do not prefer this way), since rST files with some Sphinx syntaxes (e.g. [toctree](https://github.com/microsoft/lisa/blob/wenyi/sphinx-init/docs/run_test/run.rst)) are not shown in GitHub.

Reply:
These external links were brought because the links were ending in html and I thought the docs on GitHub must be consistent with the docs on Read the Docs. However, if we have a specific doc branch, we can keep files with correct relative rst links in main and files for the build in doc branch.

Reply:
ok. For the README, we need the content in both places, and the links go to readthedocs. Other document should have the relative links.